src/styles/interactives.scss

```diff
@@ -137,6 +138,45 @@ div.os-justify-content-end {
   text-align: center;
 }
 
+// All Interactives + CTA Styles
+%os-submit-button-base {
+  border-radius: .5rem;
+  padding: 1rem 5rem;
+  color: #FFF;
+  font-weight: 700;
+}
+
+%os-submit-button-base:hover:not([disabled]) {
+  background-image: linear-gradient((rgba(0 0 0 / 20%))0 0);
+  box-shadow: none;
+}
+
+%os-submit-button-base:active:not([disabled]) {
+  background-image: linear-gradient((rgba(255 255 255 / 20%))0 0);
+  box-shadow: none;
+}
+
+%os-submit-button-base:disabled {
+  opacity: .5;
+  box-shadow: none;
+}
+
+.os-submit-button-default-theme {
+  @extend %os-submit-button-base;
+
+  background-color: #757575;
+  border: 1px solid #757575;
+  box-shadow: 0 .125rem .25rem 0 hsla(0deg 0% 46% / 25%);
+}
+
+.os-submit-button-style-theme {
```

Comment:
Would it be better to call this `.os-submit-button-green-theme`? The convention would then essentially look like `.os-submit-button-{themname}-theme` for if we add more in the future :man_shrugging: .

Same comment would apply to the `.os-cta-container-style-theme`.

Reply:
Funnily enough that was the first name I gave it, but wasn't sure if we wanted to keep using "style-theme".

Updated both.